frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/create-local-volume-set-step.tsx

```diff
@@ -73,135 +81,189 @@ const makeLocalVolumeSetCall = (
-      {csvLoadError || csv?.status?.phase !== 'Succeeded' ? (
-        <Alert
-          className="co-alert ceph-ocs-install__lso-install-alert"
-          variant="info"
-          title={t('ceph-storage-plugin~Local Storage Operator not installed')}
-          isInline
-        >
-          <Trans t={t} ns="ceph-storage-plugin">
-            Before we can create a StorageCluster, the Local Storage operator needs to be installed.
-            When installation is finished come back to OpenShift Container Storage to create a
-            StorageCluster.
-            <div className="ceph-ocs-install__lso-alert__button">
-              <Button type="button" variant="primary" onClick={goToLSOInstallationPage}>
-                Install
-              </Button>
-            </div>
-          </Trans>
-        </Alert>
-      ) : (
-        <>
-          <Grid className="ceph-ocs-install__form-wrapper">
-            <GridItem lg={10} md={12} sm={12}>
-              <Form noValidate={false}>
-                <LocalVolumeSetBody
-                  state={state}
-                  dispatch={dispatch}
-                  diskModeOptions={diskModeDropdownItems}
-                  allNodesHelpTxt={allNodesSelectorTxt}
-                  lvsNameHelpTxt={lvsNameSelectorTxt}
-                  taintsFilter={hasOCSTaint}
-                  storageClassName={storageClass.name}
-                />
-              </Form>
-            </GridItem>
-            <GridItem
-              lg={2}
-              lgOffset={10}
-              md={4}
-              mdOffset={4}
-              sm={4}
-              smOffset={4}
-              className="ceph-ocs-install__donut-chart"
-            >
-              <SelectedCapacity dispatch={dispatch} state={state} ns={ns} />
-            </GridItem>
-          </Grid>
-          <ConfirmationModal
-            ns={ns}
-            lvsNodes={lvsNodes}
-            state={state}
-            dispatch={dispatch}
-            setInProgress={setInProgress}
-            setErrorMessage={setError}
-            storageClassName={storageClass.name}
-          />
-          {state.chartNodes.size < MINIMUM_NODES && (
-            <Alert
-              className="co-alert ceph-ocs-install__wizard-alert"
-              variant="danger"
-              title={t('ceph-storage-plugin~Minimum Node Requirement')}
-              isInline
-            >
-              {t(
-                'ceph-storage-plugin~A minimum of 3 nodes are required for the initial deployment. Only {{nodes}} node match to the selected filters. Please adjust the filters to include more nodes.',
-                { nodes: state.chartNodes.size },
-              )}
-            </Alert>
-          )}
-          <RequestErrors errorMessage={error?.message} inProgress={inProgress} />
-        </>
-      )}
+    <ErrorHandler
+      loaded={discoveriesLoaded}
+      loadingMessage={
+        !csvLoaded
+          ? t('ceph-storage-plugin~Checking LSO operator installation')
```

Comment:
IIRC, some time back we decided not to use short-forms of operators name directly. We changes all such instances to full-forms.
```suggestion
          ? t('ceph-storage-plugin~Checking Local Storage Operator installation')
```

Reply:
yeah, good catch!